.github/workflows/lockfile.yml

```diff
@@ -11,3 +15,14 @@ jobs:
         uses: mansona/npm-lockfile-version@v1
         with:
           version: 3
+  lockfile_changes:
+    name: Lockfile changes check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out a copy of the repo
+        uses: actions/checkout@v4
+      - name: NPM Lockfile Changes
+        # The original doesn't support v3 lockfiles so we use a fork that adds support for them
+        uses: rvanvelzen/npm-lockfile-changes@main
```

Comment:
For the sake of security, maybe pin to a specific commit?

Reply:
addressed via 6a775843d49aa466f5f13e919f63b03771a334db